Bottom: 01d5c10f22910bd501a2e9e958169c862e696412
Top:    55a5fb6c5fe7a908b8a23433b4ebaac888c4de7e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 09:26:25 -0500

Add build variable for HSM host


---
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index 40d117db..af6742b2 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -68,12 +68,13 @@ RUN wget https://bootstrap.pypa.io/get-pip.py \
 
 # Install software for image signing
 ARG PTK_URI="http://127.0.0.1/610-009981-015_SW_PTK_5.3_Client_RevA.tar"
+ARG PTK_HSM_HOST="127.0.0.1"
 RUN curl ${PTK_URI} \
 |   tar -v -x -C /tmp \
 &&  alien --to-deb --install --scripts /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/SDKs/Linux64/ptkc_sdk/PTKcpsdk-5.3.0-16.x86_64.rpm \
 &&  alien --to-deb --install --scripts /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/SDKs/Linux64/ptkc_runtime/PTKcprt-5.3.0-15.x86_64.rpm \
 &&  alien --to-deb --install --scripts /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/SDKs/Linux64/network_hsm_access_provider/PTKnethsm-5.3.0-15.x86_64.rpm \
-&&  echo "ET_HSM_NETCLIENT_SERVERLIST=172.16.64.38" > /etc/default/et_hsm \
+&&  echo "ET_HSM_NETCLIENT_SERVERLIST=${PTK_HSM_HOST}" > /etc/default/et_hsm \
 &&  echo "/opt/safenet/protecttoolkit5/ptk/lib" > /etc/ld.so.conf.d/safenet_ptk.conf \
 &&  ldconfig -v \
 &&  rm -rf /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/
